content/10.peptide-ionization.md

```diff
@@ -1,7 +1,21 @@
 ## Peptide Ionization {.page_break_before}
-
-The 2002 Nobel Prize in Chemistry was awarded to partially to John Fenn and Koichi Tanaka "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules" [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 
+It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
```

Comment:
can we cite a review of different matrices that are possible?

Reply:
and also a general review of MALDI since you go into the mechanism here

Reply:
Do I also cover that MALDI was actually developed by Karas and Hillenkamp, but not considered for nobel, while its their  version that survives, not the one from Tanaka.

Reply:
I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

Reply:
> can we cite a review of different matrices that are possible?

Done

Reply:
> and also a general review of MALDI since you go into the mechanism here

Added

Reply:
> I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

Covered that in short. See if it fits.